_content/monographs/practical-advice-for-meditators_mills-laurence.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Practical Advice for Meditators"
+authors:
+  - mills-laurence
+external_url: "https://www.bps.lk/olib/wh/wh116_Khantipalo_Practical-Advice-For-Meditators.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1kEPSwBL8zktT1XkqHqQ9DJBE35UsQJU7/view?usp=sharing"
+  - "https://drive.google.com/file/d/1kurZQ5objbsMFl2auhKSlXZN2X0FY59I/view?usp=drive_link"
+formats: [pdf, epub]
+course: problems
+tags:
+  - parami
+  - brahmavihara
+  - hindrances
+  - daily-life
+year: 2008
+olid: OL13056099M
+publisher: bps
+address: "Kandy"
+series: "The Wheel Publication No. 116"
```

Comment:
```yaml
series: wheel
number: 116
```

And while technically these should go in monographs, I put the first few in booklets and so I've just continued to put them there 😅